src/components/hero/campaignHeroSubText.tsx

```diff
@@ -27,7 +27,7 @@ const CampaignHeroSubText = ({ featuredItem }: any) => {
               color: "var(--nypl-colors-ui-link-tertiary) !important",
             },
           }}
-          href="https://digitalcollections.nypl.org/about#nypl_harmful_content_statement"
+          href="/about#nypl_harmful_content_statement"
           aria-label="Learn more about harmful content"
           target="_blank"
```

Comment:
We can remove the `target="_blank"` in this link - Lana thinks this should open in the same tab on DC Facelift because the behavior in the old app is for this link to open in the same tab.

Probably a bug I introduced.